test/functionalTest/harnessFunctions.sh

```diff
@@ -90,49 +90,45 @@ function dMsg()
 
 # ------------------------------------------------------------------------------
 #
-# dbInit - 
+# brokerStart CB - 
 #
 function dbInit()
 {
-  role=$1
-  
-  host="${CB_DATABASE_HOST}"
-  if [ "$host" == "" ]
-  then
-    host="localhost"
-  fi
-  
-  port="${CB_DATABASE_PORT}"
-  if [ "$port" == "" ]
-  then
-    port="27017"
-  fi
-  
-  db=$host:$port/$1
+  role=$1  
+  db=$1
```

Comment:
All this 'host and port' was added by Leandro, and not so long ago ... (I think)
We probably need his opinion here